### DISCUSSION TITLE
Cron job always trigger late

### DISCUSSION THREAD

**Discussion by a9261020** [vercel/community#3926] [Help] [answered]
### Summary

Hi team,
I added the cron job few days ago and I faced the problem the cron job always tigger late. And the time of late will different after every new deploy. Found some discussion about cron delay. But no one is similar as mine.

### Details

I want the job always trigger at 00:00(UTC+0). So that I can get the daily weather on 08:00 am(UTC+8)

Above is my setting
<img width="923" alt="image" src="https://github.com/vercel/community/assets/17468283/10aa091d-59d4-4722-a08b-ad5ad3e578f4">

``` 
// vercel.json
{
  "rewrites": [{ "source": "/(.*)", "destination": "/api" }],
  "crons": [
    {
      "path": "/getDailyWeather",
      "schedule": "0 0 * * *"
    }
  ]
}
```

First deployment: cron job trigger at 8:01 (UTC+8)
Add some feature after
Second deployment: cron job trigger at 8:19 (UTC+8)
Add some feature after
Thrid deployment: cron job trigger at 8:31 (UTC+8)

Some troubleshooting for this. 
- Trigger it on my local: immediately
- Trigger it with prod: immediately
- Trigger it with in cron page: immediately

Not sure how to reproduce it on prod.

**Comment by amyegan** [comment, marked as answer]
Hey @a9261020. It sounds like you're running into [Hobby scheduling limits](https://vercel.com/docs/cron-jobs/usage-and-pricing#hobby-scheduling-limits).

> On the Hobby plan, we cannot assure a timely cron job invocation. For example, a cron job configured as 0 1 * * * (every day at 1 am) will trigger anywhere between 1:00 am and 1:59 am. For more accurate and timely cron job executions, upgrade to our [Pro](https://vercel.com/docs/accounts/plans/pro) plan.

**Reply by a9261020** [reply]
Oops, I missed that part. Thanks for your reply.

**Reply by mdebnam** [reply]
Is this still true? The page linked does not have that text any longer. I have a cron job that I've scheduled on the hour on the hobby plan and it's not running.

**Reply by ahmed0saber** [reply]
You can find it here:
https://vercel.com/docs/cron-jobs/usage-and-pricing#hobby-scheduling-limits
